.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -1410,6 +1409,27 @@ TODO(lxea): replace links with actual docs once merged
   - [ ] Windows Desktop
   - [ ] App Access
 
+## SSH Connection Resumption
+
+Verify that SSH works, and that resumable SSH is not interrupted across a Teleport Cloud tenant upgrade. 
+|   | Standard node | Non-resuming node | Peered node |
+|---|---|---|---|
+| `tsh ssh` | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> |
+| `tsh ssh --no-resume` | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> |
+| Teleport Connect | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> |
+| Web UI (not resuming) | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> |
+| OpenSSH (standard `tsh config`) | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> |
+| OpenSSH (adding `--no-resume`) | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> |
+
+Verify that SSH works, and that resumable SSH is not interrupted across a control plane restart (of either the root or the leaf cluster).
+
+|   | Tunnel node | Direct dial node |
+|---|---|---|
+| `tsh ssh` (local cluster) | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> |
+| `tsh ssh --no-resume` (local cluster) | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> |
```

Comment:
nit: I find "local cluster" to be kind of ambiguous.

```suggestion
| `tsh ssh` (leaf cluster) | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> |
| `tsh ssh --no-resume` (leaf cluster) | <ul><li> [ ] </ul></li> | <ul><li> [ ] </ul></li> |
```

Reply:
Updated that to "`tsh ssh`" without qualifiers and "`tsh ssh` (from a root cluster)" instead; the intended test target is accessing a node in the same cluster and a node in a leaf cluster, but it's obviously faster to test if you log in directly to the leaf and directly to the root instead.